tensorflow/examples/learn/examples_test.sh

```diff
@@ -28,20 +28,18 @@ then
   DIR="$DIR"/"$TEST_WORKSPACE"
 fi
 
-SKFLOW_EXAMPLE_BASE_DIR=$DIR/tensorflow/examples/learn
+TFLEARN_EXAMPLE_BASE_DIR=$DIR/tensorflow/examples/learn
```

Comment:
I would have gone for TF_LEARN_EXAMPLE_BASE_DIR, though the filenames in the doc are tflearn.